controllers/AuthController.js

```diff
@@ -51,36 +49,32 @@ class AuthController {
         const payload = {
           message: 'This email has been taken'
         };
-        return statusResponse.conflict(res, payload);
-      }
-      try {
-        const emailVerification = 'false';
```

Comment:
Good work @edogbosunny in making this code refactor, however I think the emailVerification should be a boolean data type with a default value set to fault in the database model. Kindly implement this and I think all should be fine here. Cheers.